scripts/serving/wrapper/preprocessor.py

```diff
@@ -124,11 +124,21 @@ def start(self):
             return
 
         # Phase 1: Extract model_format and yaml
-        yaml_data = self._extract_yaml()
-        if 'format' in yaml_data.items():
-            logger.error('model format missing')
-            return
-        format = yaml_data["format"]
+        inputs = os.getenv("INPUTS", "")
+        outputs = os.getenv("OUTPUTS", "")
+        format = os.getenv("FORMAT", "")
+        yaml_data = {}
+        if inputs == "" or outputs == "" or format == "":
```

Comment:
How abort just modified one of  "INPUTS"，"OUTPUTS" ？

Reply:
because it will use inputs & outputs & format only in model inference service

Reply:
The following code only uses the format, can't we just check if the format is empty?

Reply:
The inputs and outputs is used to generate config.pbtxt